app.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-require('newrelic');
+if(process.env.NEW_RELIC_LICENSE_KEY) require('newrelic');
```

Comment:
Awesome! It would be great to not see those errors locally.

In other parts of the application we check if `process.env.NODE_ENV` is equal to `production`. One benefit of using that same approach here is that we would always try to use New Relic on production, and if the license key isn't set on production, we would get a production error about it.